Multisig/UI/UI Library/VerifiableTextField.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  VerifiableTextField.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/20/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct VerifiableTextField: View {
+    var name: String
+    @State private var text: String = ""
```

Comment:
please group the non-private vars together and separate with a space from the private vars